aten/src/ATen/native/native_functions.yaml

```
@@ -1372,28 +1372,28 @@
 # `interpolation_mode` because it only supports Bilinear interpolation mode.
 # Nor does it take in `align_corners` because it only supports the mode
 # `align_corners = True`.
-- func: grid_sampler(Tensor input, Tensor grid, int interpolation_mode, int padding_mode, bool align_corners) -> Tensor
+- func: grid_sampler(Tensor input, Tensor grid, int interpolation_mode, int padding_mode, bool align_corners, bool pixel_coords) -> Tensor
```

Can we provide a default value for pixel_coords, according to my understanding, if we use pixel_coords, we will keep the old behavior, which means backward compatible (Please correct me if I am wrong).

If we don't provide default value, we will have issues with serialized TorchScript models which contain grid_sampler op.

---

@bnehoran please confirm if the suggested defaults (here and below) are backward compatible. 
```suggestion
- func: grid_sampler(Tensor input, Tensor grid, int interpolation_mode, int padding_mode, bool align_corners, bool pixel_coords=false) -> Tensor
```